### DISCUSSION TITLE
articles/i-am-lisa

### DISCUSSION THREAD

**Discussion by giscus** [hutusi/hutusi.github.com#184] [Comments]
# articles/i-am-lisa

丽莎说她的梦想是当一名老师也就是教师说只要是老师就行哪怕是小学教师也无所谓虽然大学老师更清闲但是但是大学老师现在需要博士文凭便意味着要花至少五年的时间获得学历然后再去学校找工作这是个问题丽莎想这个问题的时候可能正在五十层的办公室里对着透明的大玻璃看着左边是高楼右边是高楼前面还是高楼唉丽莎不禁叹气了她想自己还是考虑如何应付手头上的WORKINGPAPER吧回过头来满眼都是认真工作的女孩子她也是其中一员回到座位盯着电脑屏幕打开一个又一个文件上海的北京的天津的苏州的无锡的全国各地的分公司子公司控股公司合资公司的资金帐目啊成本核算啊会计报表啊人事工资银行存款存货折旧累计收益等等等等的数字啊都密密麻麻的像烦人的苍蝇拂呀拂不去丽莎心一横想开始吧隔壁的女孩过来问丽莎怎么用EXCEL汇总啊丽莎便耐心的教了她然后丽莎想开始从上海的着手吧有个B1打电话来了她们公司按照进公司年限分A1A2B1B2这样丽莎是A1而那个B1比她高两年所以她打电话差丽莎做事说丽莎啊把那年那公司那箱底稿那份那页文件复印出来传真给我丽莎说好的便去资料库里把那年那公司那箱底稿那份那页文件找出来去复印传真过去了然后丽莎回到座位上打开上海的文件准备开始做她的经理又过来说丽莎啊快把这些文件复印几份吧丽莎只要拿着文件去复印正巧复印机又坏了丽莎又动手拆拆装装给修好了复印好文件送给经理回来想这下没人打搅了吧正打算开始做时一个法国人也是她们的经理来说丽莎呀跟我一起送文件去客户那里给我当翻译当然法国人跟她说的是英文不是法文也不是中文丽莎说好啊当然丽莎对他说的也是英文不是中文也不是法文法国人扛大箱子丽莎扛小箱子都是文件还是很沉不过丽莎力气也大就给送到了客户那里其实丽莎挺不愿见客户的因为她们的很多客户都不礼貌对她们的态度很不好丽莎说她们的有些客户很凶很不讲理而且喜欢骂人有时连她们的经理都会被骂哭丽莎说她们的客户总这么问她叫什么名字进公司多久了是不是上海人啊然后就没好态度了总之这一天就这样过去了回到办公室只好加班做了不过好在有大部分同事陪她一起加班大家都在忙啊忙啊有时连饭都顾不上吃觉也顾不上睡有时一天只能睡两三个小时丽莎部门有个女孩子从星期五早上一直工作到星期六下午大家都这么辛苦丽莎在进公司前就知道四大是很辛苦很辛苦经常加班的可是没有想到加班会到凌晨三四点的更没想到有时会每天加班到两三点的更没想到还要通宵的更没想到通宵后第二天还要正常上班的更没想到有无穷无尽的事情需要做总是加班啊加班啊加班啊丽莎说这样好累每天很晚回到家里的时候还觉得累睡到床上才能轻松点丽莎以前是个喜欢睡懒觉的女孩子可是现在睡觉根本是个奢侈品那是比她们公司楼下的路易威登香奈儿还奢侈还奢侈的那些鞋子摆在橱窗里丽莎想也不比脚上的鞋子好看嘛丽莎就是这么没有太多奢求的女孩子可是现在只奢求有多一点的睡眠时间了丽莎睡着在梦里还想着明天早上起早挤地铁转公交赶到公司看见杰西卡露西娅朱丽娅说嗨我是丽莎呀

https://hutusi.com/articles/i-am-lisa

**Comment by webersongao** [comment]
大佬，分分段吧。。。。。。哈哈哈哈哈🐶

**Reply by hutusi** [reply]
哈哈，以前尝试的一种写作风格~~
